app/src/main/kotlin/com/wire/android/di/ActivityModule.kt

```diff
@@ -0,0 +1,20 @@
+package com.wire.android.di
+
+import android.app.Activity
+import android.content.Context
+import com.wire.android.util.keyboard.KeyboardInsetsProvider
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.components.ActivityComponent
+import dagger.hilt.android.qualifiers.ApplicationContext
+
+@Module
+@InstallIn(ActivityComponent::class)
+object ActivityModule {
+
+    @Provides
+    fun providesKeyboardInsetsProvider(@ApplicationContext context: Context, activity: Activity) =
```

Comment:
This seems to be leaking some context into the app as shown in the following memory heap dump error 
![imagen](https://user-images.githubusercontent.com/2468164/162991452-245b501b-2f2c-41be-8570-a3a027c2fdb3.png)